src/qt/locale/bitcoin_nl.ts

```diff
@@ -630,27 +623,27 @@
     </message>
     <message>
         <source>Quantity:</source>
-        <translation>Kwantiteit</translation>
+        <translation>Kwantiteit:</translation>
     </message>
     <message>
         <source>Bytes:</source>
         <translation>Bytes:</translation>
     </message>
     <message>
         <source>Amount:</source>
-        <translation>Bedrag:</translation>
+        <translation>Brutobedrag:</translation>
```

Comment:
Could i have some context where this would be seen? 
Brutobedrag = Gross amount.